apps/s_server.c

```diff
@@ -952,6 +952,8 @@ const OPTIONS s_server_options[] = {
     {"anti_replay", OPT_ANTI_REPLAY, '-', "Switch on anti-replay protection (default)"},
     {"no_anti_replay", OPT_NO_ANTI_REPLAY, '-', "Switch off anti-replay protection"},
     {"http_server_binmode", OPT_HTTP_SERVER_BINMODE, '-', "opening files in binary mode when acting as http server (-WWW and -HTTP)"},
+    {"no-CANames", OPT_NOCANAMES, '-',
```

Comment:
Not minus but underscore?

Reply:
Also I'd put all in lower case. I do not think the CAfile no-CAfile example is a good thing to follow.

Reply:
Using no-CA{file,path,store} is common -- 27 cases, so please use that convention.

Reply:
Well, this option has a different scope so I'm not sure that a convention is relevant here.

Reply:
you are correct and I was wrong.